### ISSUE TITLE
Install bug with phantomjs/casperjs

### ISSUE THREAD

**Issue by jakesandlund** [witheve/Eve#240]
I got this when installing on a Mac (10.9):

```
* Compiling Editor...
typings/casperjs/casperjs.d.ts(6,1): error TS6053: File 'typings/phantomjs/phantomjs.d.ts' not found.
typings/casperjs/casperjs.d.ts(33,44): error TS2304: Cannot find name 'ClipRect'.
typings/casperjs/casperjs.d.ts(36,38): error TS2304: Cannot find name 'ClipRect'.
typings/casperjs/casperjs.d.ts(171,9): error TS2304: Cannot find name 'WebPage'.
```

Then I did a:
```
npm install -g tsd
cd ui
tsd install phantomjs
```

and it worked!

**Comment by joshuafcole** [comment]
Thanks for the report, it was caused by me forgetting to commit the phantomjs typings, and was fixed by #238.

**Comment by jakesandlund** [comment]
:+1:
